src/t_hash.c

```diff
@@ -2178,7 +2334,7 @@ void hdelCommand(client *c) {
         checkType(c,o,OBJ_HASH)) return;
 
     for (j = 2; j < c->argc; j++) {
-        if (hashTypeDelete(o,c->argv[j]->ptr)) {
+        if (hashTypeDelete(o,c->argv[j]->ptr, 1)) {
```

Comment:
```suggestion
        if (hashTypeDelete(o,c->argv[j]->ptr,1)) {
```

Reply:
In this file there are more spaces after comma. Therefore I think we should strive to push spaces.... until we shall have  code styling convetnion.

Reply:
we have two styles, a comma followed by a space, and one without, but not mixing the two styles in a function call looks miserable.